notebooks/oracle2postgres/OracleToPostgres_notebook.ipynb

```diff
@@ -0,0 +1,967 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17131137-9cd1-41a0-a942-52c3fa9f34e6",
+   "metadata": {},
+   "source": [
+    "### Step 3.3 POSTGRES Parameters\n",
+    "- POSTGRES_HOST : POSTGRES instance ip address\n",
+    "- POSTGRES_PORT : POSTGRES instance port\n",
+    "- POSTGRES_USERNAME : POSTGRES username\n",
+    "- POSTGRES_PASSWORD : POSTGRES password\n",
+    "- POSTGRES_DATABASE : name of database that you want to migrate\n",
+    "- POSTGRES_SCHEMA : name of schema that you want to migrate\n",
+    "- OUTPUT_MODE : Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to overwrite)\n",
+    "- BATCH_SIZE : JDBC output batch size. Default set to 1000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77dd8c35-8c55-40e2-ae5b-9a7e6a13bf50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "POSTGRES_HOST=\"<host>\"\n",
+    "POSTGRES_PORT=\"<port>\"\n",
+    "POSTGRES_USERNAME=\"<username>\"\n",
+    "POSTGRES_PASSWORD=\"<password>\"\n",
+    "POSTGRES_DATABASE=\"<database>\"\n",
+    "POSTGRES_SCHEMA=\"<schema>\"\n",
+    "JDBCTOJDBC_OUTPUT_MODE=\"<modeoverwrite>\" # one of append/overwrite/ignore/errorifexists\n",
+    "JDBCTOJDBC_OUTPUT_BATCH_SIZE=\"1000\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "166b1536-d58e-423b-b3c2-cc0c171d275e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Step 3.4 Notebook Configuration Parameters\n",
+    "Below variables should not be changed unless required\n",
+    "- ORACLE_URL : Oracle Python URL\n",
+    "- JDBC_DRIVER : JDBC driver class\n",
+    "- JDBC_URL : Oracle JDBC URL\n",
+    "- JDBC_FETCH_SIZE : Determines how many rows to fetch per round trip\n",
+    "- MAIN_CLASS : Dataproc Template Main Class\n",
+    "- WORKING_DIRECTORY : Python working directory\n",
+    "- PACKAGE_EGG_FILE : Dataproc Template distributio file\n",
+    "- PIPELINE_ROOT : Path to Vertex AI pipeline artifacts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6f0f037-e888-4479-a143-f06a39bd5cc1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "ORACLE_URL = \"oracle://{}:{}@{}:{}?service_name={}\".format(ORACLE_USERNAME,ORACLE_PASSWORD,ORACLE_HOST,ORACLE_PORT,ORACLE_DATABASE)\n",
+    "JDBC_INPUT_DRIVER = \"oracle.jdbc.OracleDriver\"\n",
+    "JDBC_INPUT_URL = \"jdbc:oracle:thin:{}/{}@{}:{}/{}\".format(ORACLE_USERNAME,ORACLE_PASSWORD,ORACLE_HOST,ORACLE_PORT,ORACLE_DATABASE)\n",
+    "MAIN_CLASS = \"com.google.cloud.dataproc.templates.main.DataProcTemplate\"\n",
+    "WORKING_DIRECTORY = \"/home/jupyter/dataproc-templates/python/\"\n",
+    "JDBC_OUTPUT_DRIVER=\"org.postgresql.Driver\"\n",
+    "JDBC_OUTPUT_URL=\"jdbc:postgresql://{0}:{1}/{2}?user={3}&password={4}\".format(POSTGRES_HOST,POSTGRES_PORT,POSTGRES_DATABASE,POSTGRES_USERNAME,POSTGRES_PASSWORD)\n",
```

Comment:
This URL is missing `&reWriteBatchedInserts=True` attribute, see the "Oracle to PostgreSQL" section of JDBC readme file for example.